app/cypress/integration/stakingtab.spec.ts

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
+/// <reference types="@testing-library/cypress" />
+/// <reference path="../support/index.d.ts" />
+
+describe('Staking tab', () => {
+  beforeEach('Navigates to Staking tab', () => {
+    cy.dataCy('Navigation tabs')
+      .contains('Staking')
+      .click()
+
+    cy.contains('Staking balance').should('be.visible')
+    cy.contains('Current Delegation').should('be.visible')
+    cy.contains('Staking and Rewards History').should('be.visible')
+    cy.contains('Delegate Stake')
+      .as('DelegateAccordion')
+      .should('be.visible')
+
+    // expand delegation card if it is hidden
+    cy.dataCy('Delegate button')
+      .as('DelegateButton')
+      .then(($button) => {
+        // @ts-ignore: Is exists for a jquery button
```

Comment:
what is the role of `jquery` in here? :thinking:

Reply:
jquery is the only way to conditionally do stuff in cypress